template.html

```diff
@@ -0,0 +1,143 @@
+<div id='success' style='display:none;'>
+	<h2>The task has been successfully saved.</h2>
+</div>
+
+<div id='finish' style='display:none;'>
+	<h2>You have successfully contributed to all the tasks in this project. Thank you for your time. </h2>
+</div>
+
+<div id='missing_check_box' style='display:none;'>
+	<h2 id='missing_check_box_phrase' style='color:red;'>All questions are mandatory.</h2>
+</div>
+
+<div id='working_page'>
+	<h1 style="text-align:center;" id="statement">
+		
+	</h1>
+	<div id="document">
+		<div id="loading" class="alert alert-info">
+			<strong>Loading the PDF file...</strong>
+		</div>
+		<canvas id="the_canvas" style="border: 1px solid black; display:none;"/>
+	</div>
+
+	<div id="questions">
+	<div>
+</div>
+
+<script src="http://mozilla.github.com/pdf.js/build/pdf.js"></script>  
```

Comment:
[Hotlinking](http://en.wikipedia.org/wiki/Inline_linking) is bad :-) We should host this ourselves : since it is a git repository, I think this is a good use case for a [git submodule](http://git-scm.com/docs/git-submodule).
[EDIT] Also it might cause [CORS](http://en.wikipedia.org/wiki/Cross-origin_resource_sharing) problems.